packages/caliper-core/lib/worker/tx-observers/prometheus-push-tx-observer.js

```diff
@@ -111,7 +111,9 @@ class PrometheusPushTxObserver extends TxObserverInterface {
      * @private
      */
     async _sendUpdate() {
-        this.prometheusPushGateway.pushAdd({jobName: 'workers'}, function(err, _resp, _body) {
+        this.prometheusPushGateway.pushAdd({jobName: 'workers', groupings: {
+            'workerIndex': this.workerIndex,
```

Comment:
We should also include in the groupings the round number so we can distinguish rounds and I think the job name should be more explicit
```suggestion
this.prometheusPushGateway.pushAdd({jobName: 'caliper-worker', groupings: {workerIndex: this.workerIndex, roundIndex: this.currentRound }}
```

We should also remove these lines in this file as well
```javascript
        this.defaultLabels.workerIndex = this.workerIndex;
        this.defaultLabels.roundIndex = this.currentRound;
```
In the constructor and activate as we now have groupings for them, this will require some minor updates to the unit tests